src/config-import.js

```diff
@@ -215,6 +285,53 @@ async function findPresetsFile(entries) {
   return { presets, fields }
 }
 
+/**
+ * @param {ReadonlyArray<Entry>} entries
+ */
+async function findTranslationsFile(entries) {
+  const translationEntry = entries.find(
+    (entry) => entry.filename === 'translations.json'
+  )
+
+  if (!translationEntry) return
+
+  /** @type {unknown} */
+  let result
+  try {
+    result = await json(await translationEntry.openReadStream())
+  } catch (err) {
+    throw new Error('Could not parse translation.json')
+  }
+  assert(isRecord(result), 'Invalid translations.json file')
+  return parseTranslations(result)
+}
+
+/**
+ * @param {TranslationsFile} translations
+ * @returns {Translations}
+ */
+function parseTranslations(translations) {
+  /** @type {Translations} */
+  const translatedLangs = {}
+  for (let [lang, translationByLang] of Object.entries(translations)) {
+    const langTranslation = { presets: {}, fields: {} }
+    if (!isRecord(translationByLang))
+      throw new Error('invalid translation lang object')
+    for (let [docType, translationByDocType] of Object.entries(
+      translationByLang
+    )) {
+      if (!isRecord(translationByDocType))
+        throw new Error('invalid translation docType object')
+      if (!(docType === 'presets' || docType === 'fields')) continue
+
+      langTranslation[docType] = translationByDocType
+    }
+    if (Object.keys(langTranslation).length === 0) continue
```

Comment:
`Object.keys(langTranslation).length` will always be nonzero because of line 317 above. What's the intent of this line?